sdcm/cluster.py

```diff
@@ -2346,6 +2346,13 @@ def get_test_keyspaces(self):
         out = self.run_cqlsh(node=self.nodes[0], cql_cmd='select keyspace_name from system_schema.keyspaces')
         return [ks.strip() for ks in out.split('\n')[3:-3] if 'system' not in ks]
 
+    def get_test_tables(self, keyspace=None):
```

Comment:
I think we should have something like this already.

Reply:
I only found get_test_keyspaces()